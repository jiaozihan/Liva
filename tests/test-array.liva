class test {
	void main() {
		float[] a = new float[10];
		int[] b = new int[10];
		int i;
		a[0] = 1.0;
		b[0] = 1;
		for(i = 1; i < 10; i = i + 1)
		{
			a[i] = a[i - 1] + 1.0;
			b[i] = b[i - 1] + 1;
		}
        for(i = 0; i < 10; i = i + 1)
			print("a[",i,"]"," = ",a[i]," , ", "b[",i,"]"," = ",b[i], "\n");
	}
}