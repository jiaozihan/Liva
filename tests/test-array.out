a[0] = 1.000000 , b[0] = 1
a[1] = 2.000000 , b[1] = 2
a[2] = 3.000000 , b[2] = 3
a[3] = 4.000000 , b[3] = 4
a[4] = 5.000000 , b[4] = 5
a[5] = 6.000000 , b[5] = 6
a[6] = 7.000000 , b[6] = 7
a[7] = 8.000000 , b[7] = 8
a[8] = 9.000000 , b[8] = 9
a[9] = 10.000000 , b[9] = 10